src/renderer/entities/signatory/model/signatory.ts

```diff
@@ -1,4 +1,8 @@
-import { Contact } from '../../contact/model/contact';
-import { PartialBy } from '@renderer/domain/utility';
+import { AccountId, Address } from '@renderer/domain/shared-kernel';
 
-export type Signatory = PartialBy<Contact, 'name'>;
+export type Signatory = {
```

Comment:
Why you decided to rewrite it like that?

Reply:
I wanted to remove dependancy from `Contact` (FSD prohibits cross-entity imports)
Similar entities not always should be collided into 1 entity, they might grow in different directions in future.
`Signatory` is relevant in terms of multisig operations, whether `Contact` represents and acts more like `Account` with additional properties or different available actions.

Reply:
> [Копипаста в коде не всегда зло](https://bespoyasov.ru/blog/copy-paste/), это инструмент. Сделайте две почти одинаковые сущности, посмотрите, как они себя ведут в реальности, понаблюдайте за ними. В какой-то момент вы заметите, что они либо стали очень разными, либо они действительно отличаются лишь одним полем. Смёржить две похожие сущности в одну — проще, чем обмазывать код условиями проверки под каждый возможный вариант

[Link](https://bespoyasov.ru/blog/clean-architecture-on-frontend/#%D1%87%D1%82%D0%BE-%D0%B2%D0%BD%D0%B0%D1%81%D1%82%D0%BE%D1%8F%D1%89%D0%B8%D1%85-%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B0%D1%85-%D0%BC%D0%BE%D0%B6%D0%B5%D1%82-%D0%B1%D1%8B%D1%82%D1%8C-%D1%81%D0%BB%D0%BE%D0%B6%D0%BD%D0%B5%D0%B5)